public/app/plugins/datasource/cloudwatch/__mocks__/AnnotationQueryRunner.ts

```diff
@@ -0,0 +1,41 @@
+import { of } from 'rxjs';
+
+import { CustomVariableModel, DataQueryRequest } from '@grafana/data';
+import { getBackendSrv, setBackendSrv } from '@grafana/runtime';
+import { TemplateSrv } from 'app/features/templating/template_srv';
```

Comment:
should this pull from the package instead of directly within grafana?

Reply:
Unfortunately, grafana/runtime does not expose the class so to be able to instantiate it like this we need to import it directly. :(